photon-client/src/components/dashboard/tabs/AprilTagTab.vue

```diff
@@ -25,7 +25,7 @@ const interactiveCols = computed(
     <cv-select
       v-model="currentPipelineSettings.tagFamily"
       label="Target family"
-      :items="['AprilTag Family 36h11', 'AprilTag Family 25h9', 'AprilTag Family 16h5']"
+      :items="['AprilTag 36h11 (6.5in)', 'AprilTag 25h9 (6in)', 'AprilTag 16h5 (6in)']"
```

Comment:
can we just nuke 25h9

Reply:
Yes, but the enum indexes will be broken for people using 2023 settings